index.ts

```diff
@@ -1,3 +1,4 @@
+import { throttle } from 'lodash';
```

Comment:
```suggestion
import {throttle} from 'lodash';
```

[SHOULD] スペースなどのスタイルが他の記述箇所と合うようにしてください。

Reply:
:ryokai::bow: